prover/src/utils.rs

```diff
@@ -0,0 +1,159 @@
+use std::fs::File;
```

Comment:
please move utils.rs to the example directory, and move the deps `plonky2x` to dev-dependencies.